installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc

```diff
@@ -20,12 +20,29 @@ Use the following procedures to install an {product-title} cluster using the Age
 
 The following procedures deploy a single-node {product-title} in a disconnected environment. You can use these procedures as a basis and modify according to your requirements.
 
+// Downloading the Agent-based Installer
 include::modules/installing-ocp-agent-download.adoc[leveloffset=+2]
 
+// Creating the preferred manifest files
+include::modules/installing-ocp-agent-manifests.adoc[leveloffset=+2]
+
+[id="installing-ocp-agent-opt-manifests_{context}"]
+=== Optional: creating additional manifest files
+
+You can create additional manifests, such as ZTP manifests, to further configure your cluster beyond the configurations available in the `install-config.yaml` and `agent-config.yaml`.
```

Comment:
This is not really the way I would characterise it. The ZTP manifests (in the `cluster-manifests` directory) are an _alternative_ to using `install-config.yaml` (and optionally `agent-config.yaml`. In some respects they do provide finer-grained control over the cluster than the install-config, but in practice you would probably generate them from the install-config before editing. The long-term goal is that you'd be able to take ZTP manifests from a hub cluster and recreate the same deployment using the agent-based installer. For now though, they are most useful going in the other direction (generate ZTP manifests from install-config using `openshift-install agent create cluster-manifests` and then use MCE instead of the agent-based installer to create the cluster).

Other manifests go in the `openshift` directory and provide the user a way to add extra manifests (e.g. `MachineConfig`s) to the cluster during installation.